lib/lightning/oban_manager.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Lightning.ObanManager do
+  @moduledoc """
+  The Oban Manager
+  """
+  require Logger
+
+  alias Lightning.Repo
+  alias Lightning.AttemptRun
+  alias Lightning.Invocation
+
+  @doc """
+  handles oban events
+  """
+  def handle_event([:oban, :circuit, :open], _measure, meta, _pid),
+    do: Logger.info("Circuit open #{inspect(meta, pretty: true)}")
+
+  def handle_event([:oban, :circuit, :trip], _measure, meta, _pid) do
+    Logger.error("Circuit tripped with #{inspect(meta, pretty: true)}")
+
+    context = Map.take(meta, [:name])
+
+    Sentry.capture_exception(meta.error,
+      stacktrace: meta.stacktrace,
+      message: meta.message,
+      extra: context,
+      tags: %{type: "oban"}
+    )
+  end
+
+  def handle_event([:oban, :job, :exception], measure, meta, _pid) do
+    Repo.get!(AttemptRun, meta.args["attempt_run_id"])
```

Comment:
We have more than one module that uses `Oban.Worker`. So if `Lightning.Accounts` for example crashes this function will also fail (since `"attempt_run_id"` won't be in `meta.args`.

Please add a condition to check the error is from `Pipeline` before updating the Run.

See: https://github.com/sorentwo/oban#reporting-errors

From what I can see here:

```
meta.job
|> Map.take([:id, :args, :meta, :queue, :worker])  # <== I'm guessing worker will be Lightning.Pipeline
```

_NOTE_: I see the Oban README says `meta.job` and we use just `meta`. Can you please double check which one is correct.

Reply:
We do still want to do the rest of the Sentry and logging stuff no matter the worker.

Reply:
Fixed

Reply:
For `meta.job` vs `meta` I have double checked and the correct map to use is `meta.job`.

Reply:
I say the correct map to use is `meta.job` because `:queue` doesn't exist in `meta` but is inside `meta.job`